content/200-concepts/100-components/01-prisma-schema/index.mdx

```diff
@@ -195,19 +195,19 @@ datasource db {
 }
 ```
 
-In version 2.7.0 and later, you can expand variables using the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products. such as Heroku):
+You can use the `env()` function in the following places:
+
+- A datasource url
+- Generator binary targets
+
+In version 2.7.0 and later, you can expand any variables stored in the `.env` file. Use the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products such as Heroku):
```

Comment:
Honestly I think this information is just wrong here. This has nothing to do with accessing env vars from the Prisma schema. This is functionality we have only for the `.env` file and also only works there. It should be in some docuemntation about our `.env` usage and not here.

Reply:
I removed it and made the more info link clearer

Reply:
Makes sense to clean it up like this